addon/components/bsy-button.js

```diff
@@ -4,31 +4,32 @@ import layout from '../templates/components/bsy-button';
 
 export default Component.extend({
   layout,
-  tagName: "button",
-  classNames: "btn",
-  classNameBindings: ["typeClassName", "loadingClassName"],
+
+  tagName: 'button',
+  classNames: 'btn',
+  classNameBindings: ['typeClassName', 'loadingClassName'],
   isLoading: false,
-  loadingLabel: "Loading...",
+  loadingLabel: 'Loading...',
 
   typeClassName: computed('type', function() {
-    if (this.get('type') === "submit") {
-      return "btn-primary";
+    if (this.type === 'submit') {
```

Comment:
This and the CP below could return a ternary conditional or the `else` branch is unnecessary.